yt/yt/server/lib/job_proxy/config.h

```diff
@@ -228,6 +228,9 @@ class TJobProxyInternalConfig
     //! For testing purposes only.
     bool DoNotSetUserId;
 
+    // Forward variables from job proxy environment to user job.
+    bool ForwardAllEnvironmentVariables;
```

Comment:
And why exactly it should be in dynamic config? =)

Effect of this option depends on chosen job-environment which part of static config.

Reply:
All dynamizable options should be in dynamic config.
But if there are no dynamic configs for different environments, ok, but we should implement it later.

Reply:
I agree with Konstantin here, since for CRI this option can be dynamic and I see no reason to keep it static, it won't make code more complicated in any sence.